elastic_datashader/tilegen.py

```diff
@@ -637,6 +637,9 @@ def generate_nonaggregated_tile(
         metrics = {"over_max": False}
 
         if render_mode == "ellipses":
+            if z < config.ellipse_render_min_zoom:
+                img = gen_overlay(gen_empty(tile_width_px, tile_height_px), color=(128, 128, 128, 128))
```

Comment:
This is where, if the zoom level requested by the user is greater than the current minimum, a box with grey stripes (gen_overlay?) will be created and returned instead?